### ISSUE TITLE
Automate build of the menus?

### ISSUE THREAD

**Issue by mrshirts** [livecomsjournal/livecomsjournal.github.io#44]
Currently, the menus are built running a script manually.  This can/should probably be automated on code check-in with TravisCI.

**Comment by davidlmobley** [comment]
Yes. We should also think about whether there are other checks we should do at the same time, e.g. we could potentially find some library which has Travis run a link checker on all the pages.

**Comment by ptmerz** [comment]
I'm happy to look into that - not a lot of experience with Travis-CI, but that should become handy for another project, so I'm happy to gain some!

**Comment by davidlmobley** [comment]
Cool, @ptmerz . It's immensely useful for code development, but should also work in a pinch for this too. Feel free to just copy what we've done in some of our repos, e.g. github.com/mobleylab/solvationtoolkit or github.com/open-forcefield-group/openforcefield, where we're basically just copying what the Chodera lab has done in most of their repos.
